packages/abstract-core/src/native/version_control.rs

```diff
@@ -101,6 +135,13 @@ pub enum QueryMsg {
         limit: Option<u8>,
         yanked: bool,
     },
+    /// Returns [`NamespaceListResponse`]
+    #[returns(NamespaceListResponse)]
+    NamespaceList {
+        start_after: Option<String>,
+        limit: Option<u8>,
+        account_id: Option<AccountId>,
```

Comment:
We should follow the same convention with `NamespaceFilter` as in the `ModuleFilter`. This would include the `account_id`